js/clinicalTimeline.js

```diff
@@ -261,6 +284,121 @@ window.clinicalTimeline = (function(){
   }
 
   /**
+   * Hanles the drawing and panning of the overviewAxis
+   */
+  function handleOverviewAxis() {
+    var tickTemp = getTickValues(minDays, maxDays, "months");
+    var minDayTick = tickTemp[0];
+    var maxDayTick =  tickTemp[tickTemp.length-1];
+    var zoomedWidth = chart.width();
+
+    var overview = d3.select(divId).append("svg")
+      .attr("height", 75)
+      .attr("width", overviewAxisWidth)
+      .attr("class", "overview");
+
+    var xScaleOverview = d3.time.scale()
+      .domain([minDayTick, maxDayTick])
+      .range([0 , overviewAxisWidth]);
+
+    var xScaleOverviewZoomed = d3.time.scale()
+      .domain([0, -zoomedWidth])
+      .range([0 , overviewAxisWidth]);
+
+    var xScaleRectangle = d3.time.scale()
+      .domain([0, overviewAxisWidth])
+      .range([0 , -zoomedWidth]);
+
+    var overviewAxis = d3.svg.axis().scale(xScaleOverview).orient("bottom")
+      .tickFormat(function(d) { 
+        return formatTime(daysToTimeObject(d.valueOf()), "months");
+      })
+      .ticks(tickTemp.length)
+      .tickSize(3)
+      .tickPadding(4);
+
+     var overviewAxisDuplicate = d3.svg.axis().scale(xScaleOverview).orient("top")
```

Comment:
What is the overviewAxisDuplicate for exactly? Could you add some comments to explain?